src/main/java/seedu/address/model/task/repeatable/Event.java

```diff
@@ -71,7 +71,9 @@ public boolean equals(Object other) {
 
         Event otherEvent = (Event) other;
         return otherEvent.getDescription().equals(getDescription())
-                && otherEvent.getIsDone().equals(getIsDone());
```

Comment:
I think this is a complicated boolean expression as code quality says about it. You can choose to keep that or separate them 1 by 1